### DISCUSSION TITLE
How to add Quasar extension for dotenv

### DISCUSSION THREAD

**Discussion by jmorganh** [quasarframework/quasar#13062] [Quasar App Extensions] [answered]
I used the "quasar ext add @quasar/qenv"
And then when I run "quasar dev"

I get:
 Error. This command must be executed inside a Quasar v1+ project folder.

I am in the root folder of my app

**Comment by metalsadman** [comment, marked as answer]
that package is deprecated see https://quasar.dev/quasar-cli-vite/handling-process-env#using-dotenv

**Comment by jmorganh** [comment]
Thank you for your response
That worked, after I found so many options I was down a rabbit hole and
confused.
Thanks
Morgan
